src/apps/statements/service/storeStatements/queriables/getMetadataFromStatement/getBooleanMetadata.ts

```diff
@@ -0,0 +1,23 @@
+import { get, has } from 'lodash';
+
+import ActivityInteractionType from '../../../../models/ActivityInteractionType';
+import Statement from '../../../../models/Statement';
+
+export const getBooleanMetadata = (statement: Statement)
+  : {readonly [key: string]: any} => {
+  if (
+    !(
+      get(
+        statement.object,
+        ['definition', 'interactionType'],
+      ) === ActivityInteractionType.TRUE_FALSE
+      && has(statement, ['result', 'response'])
+    )
+  ) {
+    return {};
+  }
+
+  return {
+    'https://learninglocker.net/true-false-response': get(statement, ['result', 'response']),
```

Comment:
Hey @PrinceWaune, Ryan here, we need to replace the dot here with "&46;" because Mongo does not allow dots in keys so we also swap them out for the HTML code.
```suggestion
    'https://learninglocker&46;net/true-false-response': get(statement, ['result', 'response']),
```